### PR TITLE
[Utility] Chains param name - minor bug fix

### PR DESCRIPTION
**Pull request by Olshansk**
## Description

Fixing the param name used in one of the functions in `utility/actor.go`.

## Issue

Bug found ad-hoc and small enough that not creating a separate issue to track it.

## Type of change

Please mark the relevant option(s):

- [ ] New feature, functionality or library
- [x] Bug fix
- [ ] Code health or cleanup
- [ ] Major breaking change
- [ ] Documentation
- [ ] Other 

## List of changes

- Fix the param named used in `func (u *UtilityContext) GetMaxChains` from `Stake` to Chains
- Tentative discussion: should `FishermanMaxChainsParamName` be removed? 

## Testing

- [x] `make develop_test`
- [ ] [LocalNet](https://github.com/pokt-network/pocket/blob/main/docs/development/README.md) w/ all of the steps outlined in the `README`

## Required Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have tested my changes using the available tooling
- [ ] I have updated the corresponding CHANGELOG

### If Applicable Checklist
- [ ] I have updated the corresponding README(s); local and/or global
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added, or updated, [mermaid.js](https://mermaid-js.github.io) diagrams in the corresponding README(s)
- [ ] I have added, or updated, documentation and [mermaid.js](https://mermaid-js.github.io) diagrams in `shared/docs/*` if I updated `shared/*`README(s)

### CODE REVIEW

File: utility/types/gov.go [hunk 1]
```diff
@@ -18,7 +18,8 @@ const (
 	ServiceNodeMaxPauseBlocksParamName     = "service_node_max_pause_blocks"
 	ServiceNodesPerSessionParamName        = "service_nodes_per_session"
 
-	FishermanMinimumStakeParamName       = "fisherman_minimum_stake"
+	FishermanMinimumStakeParamName = "fisherman_minimum_stake"
+	// DISCUSS: Should this parameter be removed?
```
Comment (Olshansk) [inline utility/types/gov.go:22]: @andrewnguyen22 

The specification (https://github.com/pokt-network/pocket-network-protocol/tree/main/utility) only has relay chain IDs for application and service nodes and doesn't have one for fisherman.

What is the use of `FishermanMaxChainsParamName`?

Comment (andrewnguyen22) [reply utility/types/gov.go:22]: https://github.com/pokt-network/pocket/pull/303#discussion_r998776679

**Review by andrewnguyen22 — CHANGES_REQUESTED** [review]
I'm okay with leaving the DISCUSS if you are https://github.com/pokt-network/pocket/pull/303#discussion_r999653010

File: utility/types/gov.go (hunk 1 above)
Comment (andrewnguyen22) [reply utility/types/gov.go:22]: https://github.com/pokt-network/pocket/pull/303#discussion_r999653010

**Comment by Olshansk** [comment]
> I'm okay with leaving the DISCUSS if you are [#303 (comment)](https://github.com/pokt-network/pocket/pull/303#discussion_r999653010)

I left the discuss item but linked to github.com/pokt-network/pocket-network-protocol/issues/18 in its place as well.

**Review by andrewnguyen22 — APPROVED** [review]

**Pull request merged**